src/website/src/app/documentation/demos/datagrid/filtering/examples.ts

```diff
@@ -122,12 +122,12 @@ class ColorFilter implements ClrDatagridFilterInterface<User> {
   Will search into user.creation that in this case is 'Date' object
   and not the result of the pipe 'Jan 6, 2018' 
 -->
-<clr-dg-cell [clrDgField]="'creation'">{{ user.creation | date }}</clr-dg-cell>
+<clr-dg-column [clrDgField]="'creation'">{{ user.creation | date }}</clr-dg-column>
```

Comment:
Here the problem is more complicated. It's that this whole code example is NOT directly related to how you use the component, but illustrates a situation where custom filters may need to be used - as when you want to filter by date, but it will "try" to filter based on the js date object, instead of the representation (as shown in the comment above). But people quickly scan the examples and try to copy and use this snippet which is wrong on many levels. I'd either remove the `[clrDgField]="'creation'"` and leave the `clr-dg-cell`, or add 1st the column then the cell below it. This still will not help much with perceiving the snipped of "example of the example", but at least people will not be copying the wrong code.
With your change the example will still be wrong, as it will be trying to put user.creation field in the column heading :)

Reply:
Whoops -- I went through this too quickly.  I'll fix this later today.

Reply:
It only proves that what we have there is really unclear :D